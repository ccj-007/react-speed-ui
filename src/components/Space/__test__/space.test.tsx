import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Space } from '../../index';

describe('test Space component', () => {
  it('should render', () => {
    const { asFragment } = render(<Space />);
    expect(asFragment()).toMatchSnapshot();
  });
});
