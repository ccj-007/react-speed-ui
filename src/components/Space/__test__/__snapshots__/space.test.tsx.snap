// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Space component should render 1`] = `
<DocumentFragment>
  <div
    class="speed-space css-1ak7c1s-Space"
  />
</DocumentFragment>
`;
